### PR TITLE
removed refferal and linkage, validate kp violence tool not to allow …

### PR DESCRIPTION
**Pull request by bernardadero**
…future dates, added text box to specify other on referral

### CODE REVIEW

File: omod/src/main/webapp/fragments/patient/actionsPanel.gsp [hunk 1]
```diff
@@ -25,15 +25,7 @@
 
 		${ ui.includeFragment("kenyaemr", "visitMenu", [ patient: currentPatient, visit: activeVisit ])}
 
-		<ul class="float-left">
-			<h3>General Actions</h3>
-			<li class="float-left" style="margin-top: 7px">
-				<a href="${ ui.pageLink("kenyaemr", "registration/registrationSearch") }" class="float-left">
```
Comment (ojwanganto) [inline omod/src/main/webapp/fragments/patient/actionsPanel.gsp]: Is this no longer required? This is the registration link that takes one to the client search page

File: omod/src/main/webapp/fragments/patient/actionsPanel.gsp [hunk 2]
```diff
@@ -68,7 +60,7 @@
 					}
 				%>
 
-				${ ui.includeFragment("kenyaui", "widget/formLightStack", [ forms: communityOutreachForms, onFormClick: onCOFormClick ]) }
```
Comment (ojwanganto) [inline omod/src/main/webapp/fragments/patient/actionsPanel.gsp]: Please check if this change was deliberate

File: omod/src/main/webapp/fragments/patient/actionsPanel.gsp [hunk 3]
```diff
@@ -86,7 +78,7 @@
 					}
 				%>
 
-				${ ui.includeFragment("kenyaui", "widget/formLightStack", [ forms: clinicalForms, onFormClick: onCFFormClick ]) }
```
Comment (ojwanganto) [inline omod/src/main/webapp/fragments/patient/actionsPanel.gsp]: this too

File: omod/src/main/webapp/fragments/patient/actionsPanel.gsp [hunk 4]
```diff
@@ -104,7 +96,7 @@
 					}
 				%>
 
-				${ ui.includeFragment("kenyaui", "widget/formLightStack", [ forms: programLevelForms, onFormClick: onPLFormClick ]) }
```
Comment (ojwanganto) [inline omod/src/main/webapp/fragments/patient/actionsPanel.gsp]: and this

File: omod/src/main/webapp/resources/htmlforms/kp/kpReferral.html [hunk 5]
```diff
@@ -122,12 +135,24 @@
                     <td>
                         <!-- INSTITUTION REFERRED -->
                         <obs conceptId="160478AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" id="service-referred"
-                             answerConceptIds="164849AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,160546AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                             answerConceptIds="164849AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,160546AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5622"
```
Comment (ojwanganto) [inline omod/src/main/webapp/resources/htmlforms/kp/kpReferral.html]: Kindly revert this. Concept UUID is preferred to ID

File: omod/src/main/webapp/resources/htmlforms/kp/kpClinicalVisitForm.html [hunk 6]
```diff
@@ -1193,7 +1198,7 @@
                     <td width="300"><p>  <obs conceptId="165199"
                                               answerConceptIds="664,
```
Comment (ojwanganto) [inline omod/src/main/webapp/resources/htmlforms/kp/kpClinicalVisitForm.html]: Use the concept UUIDs for Positive and negative results.

File: omod/src/main/webapp/fragments/patient/actionsPanel.gsp (hunk 1 above)
Comment (bernardadero) [reply omod/src/main/webapp/fragments/patient/actionsPanel.gsp]: yeah. issue number 41.

41 | Shalin Moraa | KP | Side Menu | Remove register client from the side menu | Low | Bernard Adero | 26-Aug-19 | Resolved |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Pull request closed without merging**